examples/CMakeLists.txt

```diff
@@ -14,14 +14,22 @@ include_directories(../vendor/cxxopts/include)
 include_directories(/usr/local/include)
 include_directories(../depends)
 include_directories(../include)
+include_directories(../vendor/lmdb/libraries/liblmdb)
+include_directories(../vendor/capnproto/c++/src)
 link_directories(../vendor/s2geometry)
 link_directories(/usr/local/lib)
 link_directories(../vendor/CRoaring)
+link_directories(../vendor/capnproto)
+link_directories(../vendor/lmdb/libraries/liblmdb/)
+link_directories(../vendor/capnproto/c++/src/capnp/)
+link_directories(../vendor/capnproto/c++/src/kj/)
+link_directories(../vendor/CRoaring/src/)
+link_directories(${OPENSSL_ROOT_DIR}lib/)
```

Comment:
@nside OPENSSL_ROOT_DIR is something you are setting elsewhere to point to your homebrew/macports installed OpenSSL?